integration/kubernetes/ro-volume-exp.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+#
+# Copyright (c) 2021 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+expect -c "
+  set timeout 3
```

Comment:
This seems potentially quite low? Could you add a comment explaining why this value was chosen and/or increase it. Maybe setting this via a variable would be useful:

```bash
TIMEOUT=${TIMEOUT:-60}
    :
set timeout $TIMEOUT

```